<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="0">
           <widget class="COpenGLWidget" name="spriteWindowWidget"/>
          </item>
          <item row="3" column="1">
           <widget class="QScrollBar" name="verticalScrollBar">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QSlider" name="seekBar">
            <property name="maximum">
             <number>0</number>
            </property>
            <property name="pageStep">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="invertedControls">
             <bool>false</bool>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBothSides</enum>
            </property>
            <property name="tickInterval">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QScrollBar" name="horizontalScrollBar">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="2">
           <widget class="QGroupBox" name="groupBox_5">
            <property name="minimumSize">
             <size>
              <width>450</width>
              <height>320</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>450</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>Animation</string>
            </property>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="geometry">
              <rect>
               <x>230</x>
               <y>140</y>
               <width>211</width>
               <height>111</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="title">
              <string>Animation parameters:</string>
             </property>
             <widget class="QLineEdit" name="animationNameEdit">
              <property name="geometry">
               <rect>
                <x>80</x>
                <y>70</y>
                <width>121</width>
                <height>24</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="label_13">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>70</y>
                <width>58</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>Name:</string>
              </property>
             </widget>
             <widget class="QComboBox" name="sequenceComboBox">
              <property name="geometry">
               <rect>
                <x>80</x>
                <y>40</y>
                <width>121</width>
                <height>24</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="label_14">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>40</y>
                <width>71</width>
                <height>20</height>
               </rect>
              </property>
              <property name="text">
               <string>Sequence</string>
              </property>
             </widget>
            </widget>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>211</width>
               <height>271</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Frame measurements:</string>
             </property>
             <widget class="QLabel" name="label">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>90</y>
                <width>58</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>Width:</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_2">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>120</y>
                <width>58</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>Height:</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_3">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>150</y>
                <width>58</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>Offset X:</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_4">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>180</y>
                <width>58</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>Offset Y:</string>
              </property>
             </widget>
             <widget class="QSpinBox" name="frameWidthBox">
              <property name="geometry">
               <rect>
                <x>70</x>
                <y>90</y>
                <width>131</width>
                <height>25</height>
               </rect>
              </property>
              <property name="minimum">
               <number>-100000</number>
              </property>
              <property name="maximum">
               <number>100000</number>
              </property>
             </widget>
             <widget class="QSpinBox" name="frameHeightBox">
              <property name="geometry">
               <rect>
                <x>70</x>
                <y>120</y>
                <width>131</width>
                <height>25</height>
               </rect>
              </property>
              <property name="minimum">
               <number>-100000</number>
              </property>
              <property name="maximum">
               <number>100000</number>
              </property>
             </widget>
             <widget class="QSpinBox" name="frameYOffsetBox">
              <property name="geometry">
               <rect>
                <x>70</x>
                <y>180</y>
                <width>131</width>
                <height>25</height>
               </rect>
              </property>
              <property name="minimum">
               <number>-100000</number>
              </property>
              <property name="maximum">
               <number>100000</number>
              </property>
             </widget>
             <widget class="QSpinBox" name="frameXOffsetBox">
              <property name="geometry">
               <rect>
                <x>70</x>
                <y>150</y>
                <width>131</width>
                <height>25</height>
               </rect>
              </property>
              <property name="minimum">
               <number>-100000</number>
              </property>
              <property name="maximum">
               <number>100000</number>
              </property>
             </widget>
             <widget class="QSpinBox" name="frameXBox">
              <property name="geometry">
               <rect>
                <x>70</x>
                <y>30</y>
                <width>131</width>
                <height>25</height>
               </rect>
              </property>
              <property name="minimum">
               <number>-100000</number>
              </property>
              <property name="maximum">
               <number>100000</number>
              </property>
             </widget>
             <widget class="QSpinBox" name="frameYBox">
              <property name="geometry">
               <rect>
                <x>70</x>
                <y>60</y>
                <width>131</width>
                <height>25</height>
               </rect>
              </property>
              <property name="minimum">
               <number>-100000</number>
              </property>
              <property name="maximum">
               <number>100000</number>
              </property>
             </widget>
             <widget class="QLabel" name="label_8">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>60</y>
                <width>58</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>Top:</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_9">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>30</y>
                <width>58</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>Left:</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_10">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>210</y>
                <width>91</width>
                <height>20</height>
               </rect>
              </property>
              <property name="text">
               <string>Duration:</string>
              </property>
             </widget>
             <widget class="QSpinBox" name="frameDurationBox">
              <property name="geometry">
               <rect>
                <x>70</x>
                <y>210</y>
                <width>131</width>
                <height>25</height>
               </rect>
              </property>
              <property name="minimum">
               <number>-100000</number>
              </property>
              <property name="maximum">
               <number>100000</number>
              </property>
             </widget>
            </widget>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>290</y>
               <width>211</width>
               <height>131</height>
              </rect>
             </property>
             <property name="title">
              <string>Sequence parameters:</string>
             </property>
             <widget class="QLineEdit" name="sequenceNameEdit">
              <property name="geometry">
               <rect>
                <x>70</x>
                <y>30</y>
                <width>131</width>
                <height>24</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="label_5">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>64</y>
                <width>58</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>Offset X:</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_6">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>91</y>
                <width>58</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>Offset Y:</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_7">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>34</y>
                <width>58</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>Name:</string>
              </property>
             </widget>
             <widget class="QSpinBox" name="seqXOffsetBox">
              <property name="geometry">
               <rect>
                <x>70</x>
                <y>60</y>
                <width>131</width>
                <height>25</height>
               </rect>
              </property>
              <property name="minimum">
               <number>-100000</number>
              </property>
              <property name="maximum">
               <number>100000</number>
              </property>
             </widget>
             <widget class="QSpinBox" name="seqYOffsetBox">
              <property name="geometry">
               <rect>
                <x>70</x>
                <y>90</y>
                <width>131</width>
                <height>25</height>
               </rect>
              </property>
              <property name="minimum">
               <number>-100000</number>
              </property>
              <property name="maximum">
               <number>100000</number>
              </property>
             </widget>
            </widget>
            <widget class="QGroupBox" name="groupBox">
             <property name="geometry">
              <rect>
               <x>230</x>
               <y>20</y>
               <width>211</width>
               <height>111</height>
              </rect>
             </property>
             <property name="title">
              <string>Move sprite window</string>
             </property>
             <widget class="QPushButton" name="pushButton_3">
              <property name="geometry">
               <rect>
                <x>70</x>
                <y>40</y>
                <width>31</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>→</string>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>40</y>
                <width>31</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>←</string>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton_4">
              <property name="geometry">
               <rect>
                <x>40</x>
                <y>50</y>
                <width>31</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>↓</string>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton_5">
              <property name="geometry">
               <rect>
                <x>40</x>
                <y>30</y>
                <width>31</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>↑</string>
              </property>
             </widget>
             <widget class="QPushButton" name="addFrameButton">
              <property name="geometry">
               <rect>
                <x>110</x>
                <y>30</y>
                <width>91</width>
                <height>31</height>
               </rect>
              </property>
              <property name="text">
               <string>Add frame</string>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton_7">
              <property name="geometry">
               <rect>
                <x>110</x>
                <y>70</y>
                <width>91</width>
                <height>31</height>
               </rect>
              </property>
              <property name="text">
               <string>Remove frame</string>
              </property>
             </widget>
            </widget>
            <widget class="QPushButton" name="addSequenceButton">
             <property name="geometry">
              <rect>
               <x>230</x>
               <y>270</y>
               <width>91</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>AddSequence</string>
             </property>
            </widget>
            <widget class="QSpinBox" name="seqLoopStartBox">
             <property name="geometry">
              <rect>
               <x>300</x>
               <y>346</y>
               <width>131</width>
               <height>25</height>
              </rect>
             </property>
             <property name="minimum">
              <number>-100000</number>
             </property>
             <property name="maximum">
              <number>100000</number>
             </property>
            </widget>
            <widget class="QLabel" name="label_11">
             <property name="geometry">
              <rect>
               <x>227</x>
               <y>350</y>
               <width>61</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string>Loop start</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_12">
             <property name="geometry">
              <rect>
               <x>230</x>
               <y>380</y>
               <width>58</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>Loop end</string>
             </property>
            </widget>
            <widget class="QSpinBox" name="seqLoopEndBox">
             <property name="geometry">
              <rect>
               <x>300</x>
               <y>376</y>
               <width>131</width>
               <height>25</height>
              </rect>
             </property>
             <property name="minimum">
              <number>-100000</number>
             </property>
             <property name="maximum">
              <number>100000</number>
             </property>
            </widget>
            <widget class="QCheckBox" name="loopCheckBox">
             <property name="geometry">
              <rect>
               <x>310</x>
               <y>320</y>
               <width>83</width>
               <height>21</height>
              </rect>
             </property>
             <property name="text">
              <string>Loop</string>
             </property>
            </widget>
            <widget class="QPushButton" name="playButton">
             <property name="geometry">
              <rect>
               <x>400</x>
               <y>270</y>
               <width>31</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>&gt;</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pauseButton">
             <property name="geometry">
              <rect>
               <x>370</x>
               <y>270</y>
               <width>31</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>||</string>
             </property>
            </widget>
            <widget class="QPushButton" name="stopButton">
             <property name="geometry">
              <rect>
               <x>340</x>
               <y>270</y>
               <width>31</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>□</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QScrollBar" name="verticalScrollBar_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QScrollBar" name="horizontalScrollBar_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="COpenGLWidget" name="atlasWindowWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1061</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Sprite"/>
    <addaction name="actionSave"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSprite">
    <property name="title">
     <string>Sprite</string>
    </property>
    <addaction name="actionLoad_Image"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSprite"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Sprite">
   <property name="text">
    <string>Open Sprite</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionLoad_Image">
   <property name="text">
    <string>Load Image</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save Animation</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>COpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">openglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
